core/src/platform.h

```diff
@@ -44,35 +44,55 @@ void logMsg(const char* fmt, ...);
 void requestRender();
 
 /* If called with 'true', the windowing system will re-draw frames continuously;
- * otherwise new frames will only be drawn when 'requestRender' is called. 
+ * otherwise new frames will only be drawn when 'requestRender' is called.
  */
 void setContinuousRendering(bool _isContinuous);
 
 bool isContinuousRendering();
 
+/* Get the system path for font files
+ *  for each platform this will be different
+ */
+std::string deviceFontsPath();
+
+/* Construct font file name from font properties specified in yaml */
+std::string constructFontFilename(const std::string& _name, const std::string& _weight, const std::string& _face);
```

Comment:
I feel like `constructFontFileName` and `deviceFontsPath` can be consolidated into a single function like `getFontFilePath`, I can't see how these would be used separately and they may not be sufficient for cases like the multiple font file paths in OS X